### PR TITLE
Add SwitchingAssistantSection story

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Add SwitchingAssistantSection by using Apollo mock

https://github.com/HedvigInsurance/racoon/assets/6661511/424d5f63-5a6d-44fe-93d4-14c0d9dee588

## Justify why they are needed
Make it possible to view the component i Storybook
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2700** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2700" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2700?utm_source=stack-comment).

**Comment by gustaveen** [comment]
Maybe it makes more sense to have a story for just the `ContractCard` :D

File: apps/store/src/components/ConfirmationPage/SwitchingAssistantSection/SwitchingAssistantSection.tsx [hunk 1]
```diff
@@ -22,7 +22,13 @@ export const SwitchingAssistantSection = (props: Props) => {
           {t('CONFIRMATION_SWITCHING_SUBTITLE', { company: props.companyDisplayName })}
         </Text>
       </div>
-      {switchingContracts.length === 0 ? <CardSkeleton /> : switchingContracts.map(ContractCard)}
+      {switchingContracts.length === 0 ? (
+        <CardSkeleton />
+      ) : (
+        switchingContracts.map((switchingContract) => (
+          <ContractCard key={switchingContract.id} {...switchingContract} />
```
Comment (robinandeer) [inline apps/store/src/components/ConfirmationPage/SwitchingAssistantSection/SwitchingAssistantSection.tsx:29]: ahh nice

**Review by robinandeer — APPROVED** [review]

**Comment by gustaveen** [comment]
@gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2700).

**Pull request merged**